doctr/models/recognition/sar/tensorflow.py

```diff
@@ -302,10 +302,11 @@ def __call__(
         decoded_strings_pred = tf.strings.reduce_join(inputs=tf.nn.embedding_lookup(embedding, out_idxs), axis=-1)
         decoded_strings_pred = tf.strings.split(decoded_strings_pred, "<eos>")
         decoded_strings_pred = tf.sparse.to_dense(decoded_strings_pred.to_sparse(), default_value='not valid')[:, 0]
-        word_values = [word.decode() for word in decoded_strings_pred.numpy().tolist()]
-
-        return list(zip(word_values, probs.numpy().tolist()))
 
+        return {
+            "decoded_strings_pred":decoded_strings_pred,
+            "probs":probs,
+        }
```

Comment:
We can't change the output signature of the call method in this PR or we'll break the high-level API